docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md

```diff
@@ -0,0 +1,106 @@
+
+* In order to keep your local environment clean and untouched, we will use [Google Cloud Shell](https://cloud.google.com/shell) to run the [*gke_monitor_onboarding*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/gke/gke_monitor/azure_monitor_k8s_extension.sh) shell script against the GKE connected cluster.
+
+* Before integrating the cluster with Azure Monitor for Containers, click on the "Insights (preview)" blade for the connected Arc cluster to show how the cluster is not currently being monitored.
+
+    ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource](./01.png)
+
+    ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource Insights](./02.png)
+
+* Edit the environment variables in the script to match your environment parameters and upload it to the Cloud Shell environment. After than run it using the ```. ./azure_monitor_k8s_extension.sh``` command.
+
+    > **Note: The extra dot is due to the shell script has an *export* function and needs to have the vars exported in the same shell session as the rest of the commands.**
+
+    ![Screenshot showing GKE cluster in GCP console](./03.png)
+
+    ![Screenshot showing connection to GKE cluster in GCP console](./04.png)
+
+    ![Screenshot showing uploading the script file to cloud shell](./07.png)
+
+    The script will:
+
+  * Login to your Azure subscription using the SPN credentials
+  * Add or Update the connectedk8s and k8s-extension
+  * Create monitor k8s extension instance
+
+* Now if you can see that the monitoring is enabled once you visit Container Insights section of Azure Arc enabled kubernetes cluster in Azure.
+
+    > **Note: As the OMS start collecting telemetry from the cluster nodes and pods, it will take 5-10min for data to start show up in the Azure Portal.**
+
+* Click the "Connected Clusters" tab and see the Azure Arc connected cluster was added. Now that your cluster is being monitored, navigate trough the different tabs and sections and watch the monitoring telemetry for the cluster nodes and pods.  
+
+![Screenshot showing Monitoring console](./06.png)
+
+### Delete extension instance
+
+The following command only deletes the extension instance, but doesn't delete the Log Analytics workspace. The data within the Log Analytics resource is left intact. You can visit the Insights seciton of Azure Arc resource in Azure Portal and can see the onboarding state as not connected.
+
+```bash
+az k8s-extension delete --name azuremonitor-containers --cluster-type connectedClusters --cluster-name <cluster-name> --resource-group <resource-group>
+```
+
+ ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource Insights](./02.png)
```

Comment:
```suggestion
 ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource Insights](./02.png)
```
Change filename to _07.png_

See [this section](https://azurearcjumpstart.io/scenario_guidelines/#screenshots) in the write-up guildeines.

Reply:
fixed and let me know if there any further updates needed. Thanks!